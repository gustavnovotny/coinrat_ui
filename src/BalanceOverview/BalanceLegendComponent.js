// @flow
import React from "react"

const BalanceLegendComponent = () => {
  return <div>
    <p>
      This table <strong>ALWAYS</strong> shows real balances on the selected market.
      There is no mock market involved.
    </p>
  </div>
}

export default BalanceLegendComponent
