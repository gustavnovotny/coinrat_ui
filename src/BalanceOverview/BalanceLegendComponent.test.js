// @flow
import React from 'react'
import renderer from 'react-test-renderer'
import BalanceLegendComponent from "./BalanceLegendComponent"

it('renders correctly', () => {
  const tree = renderer.create(<BalanceLegendComponent/>).toJSON()
  expect(tree).toMatchSnapshot()
})
