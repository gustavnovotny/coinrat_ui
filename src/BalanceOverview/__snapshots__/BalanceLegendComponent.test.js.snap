// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <p>
    This table 
    <strong>
      ALWAYS
    </strong>
     shows real balances on the selected market. There is no mock market involved.
  </p>
</div>
`;
